module/data/vars.py

```diff
@@ -116,8 +128,42 @@ class TEXT_IDS(Enum):
     EXAMS_USAGE_TEXT_ID = auto()
 
 
+# Placeholder special string to be used in replace()
 PLACE_HOLDER: str = "%capybara%"
 
+
+# List of textx that are dynamic
+# The reverse is used as a translation table to bridge md and informative handlers into localization handler
+ON_DEMAND_TEXTS: dict[str, str] = {
+    "HELP_CDL_EXAMS_LINK_TOOLTIP_ID": "esami_link",
+    "HELP_CDL_CLASSES_LINK_TOOLTIP_ID": "lezioni_link",
+    "HELP_CDL_LIBRARY_TOOLTIP_ID": "biblioteca",
+    "REPRS_DMI_TOOLTIP_ID": "rappresentanti_dmi",
```

Comment:
Why put this in "ON_DEMAND_TEXTS"? So it can easily be modified from the .md file?

Reply:
md files contains only essential stuff like working hours or contacts, the message template with fixed sentences can be found inside yaml files.